LegitimateURLShortener.txt

```diff
@@ -2327,6 +2327,13 @@ $removeparam=height,domain=i-viaplay.com.akamaized.net|media.discordapp.net|tvno
 !#endif
 
 ! ——— Allowlist ——— !
+
```

Comment:
Why is there a space here?

Reply:
Sorry for the accidental addition of blank lines.

Reply:
Its ok. Errors happen